code/game/objects/items/clown_items.dm

```diff
@@ -73,6 +73,13 @@
 	icon_state = "soapsyndie"
 	cleanspeed = 5 //faster than mop so it is useful for traitors who want to clean crime scenes
 
+/obj/item/soap/omega
+	name = "omega soap"
+	desc = "The most advanced soap known to mankind."
+	icon_state = "soapomega"
+	cleanspeed = 3 //Only the truest of mind soul and body get one of these
+	uses = 301

```

Comment:
I'd put the uses at double the regular amount at least, maybe even infinite.

Reply:
it already has extremely fast cleanspeed

Reply:
Yeah, give it an infinite number of uses.

Reply:
for powerlevel consider that this still won't beat a floor buffer or a janiborg, unless all the mess is specifically under tables or on walls

Reply:
I do not plan on giving it an absurd number or infinite amount of uses.

Reply:
You will not need to use this more than 301 times and if you do just whip up a double batch in a bluespace beaker.

Reply:
give it aoe telephathic cleaning or its gonna be made once or twice then it ll turn into a lazy admins epic event loot

Reply:
fyi cleaning skill makes it so there's a chance it won't use one of the uses.